tests/pipeline/test_term.py

```diff
@@ -243,6 +288,30 @@ class SomeFactor(Factor):
 
         self.assertIsNot(orig_foobar_instance, SomeFactor())
 
+    def test_instance_non_caching_multiple_outputs(self):
+        multiple_outputs = MultipleOutputs()
```

Comment:
Might want to verify that this produces the same result if you do `MultipleOutputs(outputs=MultipleOutputs.outputs)`.